manifests/params_agent.pp

```diff
@@ -82,9 +82,8 @@
 
       ## Server block configuration
 
-      $ossec_ip                          = 'YOUR_MANAGER_IP'
-      $ossec_hostname                    = undef
-      $ossec_address                     = undef
+      $ossec_registration_ip             = undef
```

Comment:
Could we set `'127.0.0.0'` as a default value?

When building an AWS IAM that will be used in multiple environments, you want to install the Wazuh agent without registering it. In the AMI build, you cannot set the final master/worker nodes endpoints because they change from one environment to another.

Setting a default value like `'127.0.0.0'` will simplify the code that installs the Wazuh agent without registering it.

Reply:
I declared them to be `undef` to make sure users consciously set them to avoid misconfigurations, but as you stated maybe configure it to 127.0.0.0 would be a good idea, I will think about it.

I take the chance to mention that I implemented a variable to configure if the Agent will atempt to auto-register or not: `$manage_client_keys`

You can check it at:

https://github.com/wazuh/wazuh-puppet/blob/a172a11b675ce797747c294eb02260b7f98bdc4d/manifests/params_agent.pp#L15

Thanks again for contributing!